yaml/pois.yaml

```diff
@@ -1812,3 +2106,32 @@ filters:
       <<: *output_properties
       kind: attraction
       tier: 6
+
+  # if a more specific shop isn't found, then default to generic shop kind
+  - filter:
+      shop: true
+    min_zoom: 17
+    output:
+      <<: *output_properties
+      kind: shop
+  # if a more specific craft isn't found, then default to generic craft kind
+  - filter:
+      craft: true
+    min_zoom: 17
+    output:
+      <<: *output_properties
+      kind: craft
```

Comment:
Please ensure test and documentation for `craft` fallback.

Reply:
Yup, already had [documentation](https://github.com/tilezen/vector-datasource/blame/c01d34b1f76e665c902d8b5561febe6722aca795/docs/layers.md#L857) and [test](https://github.com/tilezen/vector-datasource/blob/c01d34b1f76e665c902d8b5561febe6722aca795/integration-test/1424-add-new-kinds-for-hot-icon-support.py#L222-L244) for `craft` fallback.